src/reactor.rs

```diff
@@ -98,7 +98,10 @@ impl Reactor {
                 sources: Mutex::new(Slab::new()),
                 events: Mutex::new(Vec::new()),
                 timers: Mutex::new(BTreeMap::new()),
+                #[cfg(not(target_os = "espidf"))]
                 timer_ops: ConcurrentQueue::bounded(1000),
+                #[cfg(target_os = "espidf")]
+                timer_ops: ConcurrentQueue::bounded(100),
```

Comment:
I'd rather have a `const TIMER_QUEUE_SIZE` that changes based on OS, rather than changing the value at the call site.

Reply:
Done.